tools/npm_integration_test/test_runner.js

```diff
@@ -188,8 +188,12 @@ class TestRunner {
           if (binary.startsWith('external/')) {
             binary = `../${binary.substring('external/'.length)}`;
           }
-          const runfilesBinary = runfiles.resolveWorkspaceRelative(binary);
-          binary = fs.existsSync(runfilesBinary) ? runfilesBinary : binary;
+          let runfilesBinary;
+          try {
+            runfilesBinary = runfiles.resolveWorkspaceRelative(binary);
+          } catch (e) {
+          }
+          binary = (runfilesBinary && fs.existsSync(runfilesBinary)) ? runfilesBinary : binary;
```

Comment:
NIT: This could be a bit simpler:

```suggestion
          try {
            const runfilesBinary = runfiles.resolveWorkspaceRelative(binary);
            binary = (runfilesBinary && fs.existsSync(runfilesBinary)) ? runfilesBinary : binary;
          } catch (e) {
          }
```

Reply:
Nice catch. Thanks.